### ISSUE TITLE
CloudCompare new install of Stereo 2.9.1 64x after 2.8 doesn't update path in HKEY_CLASSES_ROOT\CloudCompare with "STEREO" in pathname.

### ISSUE THREAD

**Issue by Dbhericks** [CloudCompare/CloudCompare#735]
## Current Behavior

Windows 7  After installing CloudCompare Stereo 2.9.1 , after uninstalling previous 2.8 stereo. I could not associate *.BIN file type to CloudCompare.exe.  CloudCompare did not even show up as an optional application to associate with and even selecting the "browse" and navigating to \CloudCompareStereo\CloudCompare.exe  it did not show up as an application that BIN could be associated to. 

Fix: 
Resolution 
Follow the steps below:
1. Type "regedit" in the Run line or from an elevated CMD prompt.
2. Navigate to Computer\HKEY_CLASSES_ROOT\Applications\CloudCompare and find your .exe name. 
3. Navigate under its name to shell>open>command. 

4. Under Default, make sure the application location points to the actual location of the executable.
(i.e. Change "C:ProgramFiles\CloudCompare\CloudCompare.exe" "%1"   to
 "C:ProgramFiles\CloudCompareStereo\CloudCompare.exe" "%1"  )

5. Press OK and then try and reassociate the file type as you normally would.

https://support.microsoft.com/en-us/help/2159999/sometimes-you-can-not-associate-a-program-with-an-extension-in-windows

## Your Environment
* CC Version: CloudCompareStereo 2.9.1
* OS & Version: Win7  Enterprise
Version	6.1.7601 Service Pack 1 Build 7601
* Graphics card:  
NVIDIA Quadro M1000M Driver Version	22.21.13.8607

![image](https://user-images.githubusercontent.com/38393927/39667255-9f105820-5066-11e8-82c5-e18f20e5decc.png)

**Comment by dgirardeau** [comment]
Hum, as this extension-to-application linking is managed by Windows, the CloudCompare uninstaller doesn't know about it, and it doesn't remove the association indeed. Do you suggest that the installer should remove this association? If yes, do you know how to do it with InnoSetup for instance? (looks tricky to me, because one needs to find where this association is exactly dynamically, and then remove it only if it corresponds to the uninstalled version of CloudCompare...)

**Comment by Dbhericks** [comment]
Daniel,
I think the STEREO installer needs to change the Path name in the registry (or create a new registry entry? ) that correctly points to CloudCompare.EXE in the CloudCompare\Stereo\*exe  path, not the  CloudCompare\*exe path.

I cannot offer any advice on how to fix this as I have no expertise in this area.
Perhaps it was created from the following and is an unique case?

I over last 7 months I originally installed CC ~2.8? Non-Stereo,
Then installed ~2.8 Stereo (which I believe was in the CloudCompare\CloudCompare.exe) ,
then uninstalled ~2.8 Stereo and
then installed 2.9.1 Stereo (which went into the CloudCompare\Stereo\*exe  path)

I was able to fix the registry entry manually per the image I attached and directions in the Windows knowledgebase related to programs not being registered correctly.
( I think that maybe in the rare case this happens a documented work-around would be sufficient? )
